libkbfs/bare_root_metadata_v2.go

```diff
@@ -309,9 +309,11 @@ func (md *BareRootMetadataV2) IsFinal() bool {
 // IsWriter implements the BareRootMetadata interface for BareRootMetadataV2.
 func (md *BareRootMetadataV2) IsWriter(
 	user keybase1.UID, deviceKey kbfscrypto.CryptPublicKey, _ ExtraMetadata) bool {
-	if md.ID.IsPublic() {
+	if md.ID.Type() != tlf.Private {
```

Comment:
IMO this should be `md.ID.Type() == tlf.Public`, since the teams version of this won't need to iterate through the `md.Writers` list.